code/index.html

```diff
@@ -2,11 +2,66 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kreativhang Checkout</title>
     <!-- dont forget the link to the css file -->
+    <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+  
   <body>
     <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
+    <header>  
+      <img src="img/Kreativhang_Logo.svg" alt="Kreativhang"/>
+      <h1>Freude schenken</h1>
+    </header>
+
+    <main>
+
+    <h2>Bestellung</h2>
+    <p>Geben Sie Ihre E-Mail-Adresse ein. </br>
+    An diese Adresse werden Ihnen Updates zum Bestellstatus gesendet. </p>
+
+    <form action="http://httpbin.org/anything" method="POST">
+
+    <label>
+      E-Mail
+      <input type="mail" name="mail" placeholder="hello@example.ch" required>
```

Comment:
Should be `type="email"`. Really nice with a placeholder ⭐

Reply:
When you have more than two attributes, it's nice to split them into different rows. Also, it's a convention to start with boolean attributes (attributes that can only be true or false like `required`):
```
<input
  required
  type="email"
  name="mail"
  placeholder="hello@example.ch"
/>
```